src/Components/Empty/Empty.jsx

```diff
@@ -1,19 +1,32 @@
-import "./Empty.css"
 import React from 'react';
+
 import { Navbar } from "../Navbar/Navbar";
-import { Sidebars } from "../SideBars/Sidebars";
+import { MenuBar } from "../MenuBar/MenuBar";
 import { Emptyimage } from "../../Images";
 
+import "./Empty.scss"
+
```

Comment:
name of scss or css file will be in small case and if there two words , should separate by hypen